test/e2e/common/networking.go

```diff
@@ -126,5 +126,27 @@ var _ = ginkgo.Describe("[sig-network] Networking", func() {
 				}
 			}
 		})
+
+		// Once basic tests checking for the sctp module not to be loaded are implemented, this
+		// needs to be marked as [Disruptive]
+		ginkgo.It("should function for intra-pod communication: sctp [LinuxOnly][Disruptive]", func() {
+			config := e2enetwork.NewNetworkingTestConfig(f, e2enetwork.EnableSCTP)
+			checkPodToPodConnectivity(config, "sctp", e2enetwork.EndpointSCTPPort)
+		})
+
+		// Once basic tests checking for the sctp module not to be loaded are implemented, this
+		// needs to be marked as [Disruptive]
+		// Skip until DialFromNode() supports SCTP
+		ginkgo.It("should function for node-pod communication: sctp [LinuxOnly][Disruptive]", func() {
+			ginkgo.Skip("Skipping SCTP node to pod test until DialFromNode supports SCTP")
```

Comment:
You don't need both the comment _and_ the `ginkgo.Skip`, but you do need an open issue to remind us this needs to be fixed

Reply:
done